<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookManagementDialog</class>
 <widget class="QDialog" name="BookManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="isbn_edt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ISBN_ra">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="idx_ra">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图书号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查  询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_book_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新 增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_book_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保 存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_book_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删 除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_bookinfo_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新  建</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_name_edt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>语言：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lang_edt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>版本：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ver_edt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>简介：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="brief_edt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="auth_edt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="press_edt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolButton" name="author_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>27</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="press_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>27</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_bookinfo_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保  存</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>491</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
